ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
Ok. I see the rationale. 

Yup, having it in dcos-launch, and other * invokers* is good for this change.

There is one advantage is having a logger at a single place where the modules (even the clients/invokers) can include. 

* Enforcing consistency in the log format.  
* Changing backend. for .e.g we want to publish all logs to a single place. If every client imported a common logger, only that will need change.